### ISSUE TITLE
The program does not build on Windows

### ISSUE THREAD

**Issue by ammongit** [ammongit/homesteader#1]
Ideas for solving this:
- Cygwin
- Cygwin with mingw
- mingw cross-compilation
- MSYS2

**Comment by ammongit** [comment]
I have been able to build the program on Windows using Cygwin and the mingw64-x86_64 toolchain.

Install at least the following cygwin packages:
* git
* make
* mingw64-x86_64-gcc-core
* mingw64-x86_64-winpthread
* mingw64-x86_64-ncurses
* mingw64-x86_64-libao

Then:
```bash
$ git clone https://github.com/ammongit/homesteader.git project
$ cd project/src
$ export CC=x86_66-w64-mingw32-gcc
$ make
```

The project compiles, but will not run without the appropriate DLLS. (Use `cp` if you linking the libraries fails)
```bash
$ ln /usr/x86_64-w64-mingw32/sys-root/mingw/bin/libwinpthread-1.dll .
$ ln /usr/x86_64-w64-mingw32/sys-root/mingw/bin/libao-4.dll .
$ ln /usr/x86_64-w64-mingw32/sys-root/mingw/bin/libncursesw6.dll .
```

Once you copy then into `%PATH%` or the same directory as the binary, then it will run.

So it builds. But there are some caveats. For one, it does not run in Cygwin, though this may be a limitation of Cygwin itself in how it maintains its environment. Which is fine, I'm trying to create a standalone Windows executable rather than one that requires the cygwin environment.

But as #8 points out, using CMD as our terminal causes problems. I've already patched in a direct call to the winapi to change the title, but I'd like to avoid doing that for colors too. It's probably some kind of ncurses configuration that needs to be done.
